### ISSUE TITLE
SC 3.2.6 Findable Help - Definition for "Set of Web Pages" Needs Refinement

### ISSUE THREAD

**Issue by omar-bonilla** [w3c/wcag#1436]
On Behalf of Thomson Reuters:

The definition of a “set of web pages” is too ambiguous. Providing more detail through examples of sets of web pages is required to give content authors a better understanding of the scope of this Success Criterion.

**Comment by scha14** [comment]
Draft response 

> `Set of Web pages` is defined with examples at the bottom of the [understanding document](https://w3c.github.io/wcag/understanding/findable-help#dfn-set-of-web-pages) as `collection of web pages that share a common purpose and that are created by the same author, group or organization`. 
> 
> Examples include a publication which is split across multiple Web pages, where each page contains one chapter or other significant section of the work. The publication is logically a single contiguous unit, and contains navigation features that enable access to the full set of pages.
> 
> NOTE
> Different language versions would be considered different sets of Web pages.
>
> Would more examples help in clarifying the scope?

**Comment by scha14** [comment]
New response approved by the group, same as[ issue 1427](https://github.com/w3c/wcag/issues/1427 ), so closing. 

<hr>

> I don't see how a UI component can be included in the same relative order on each page within a single page web application.
> Perhaps "single page" should be replaced with "multiple page"?

Part of the confusion comes from the original definition of web-page, which is based on the URI, so oddly (to modern thinking) an SPA which does not update the URL is a 'web page', not a set of web pages.

For this SC we have three conditions covered by 2 definitions:

- Web pages, in the traditional sense.
- SPAs with different URIs, covered by the standard definition of web pages and 'set of web pages'.
- SPA without routing (same URI), covered by the new SPA definition.

It is possible in an SPA (same URI style) could move navigation and/or help links around the layout, so the SC is intended to apply in this instance.

Please note we have added a paragraph on SPAs to the understanding document recently:
> A single page Web application (compared to a Web page) shows multiple "pages" or views of content at the same URI. If a web application uses different URIs for each view of the content, that is considered multiple Web pages because the URI changes.

With that update to the understanding document, and this explanation, the group considers this issue addressed, but please re-open if there is something about this issues that you don't consider to be addressed.

**Comment by scha14** [comment]
@alastc reopening this one as well - referring to set of web pages and scope, instead of the SPA discussion

**Comment by alastc** [comment]
Hi everyone,

The text of the [success criterion](https://raw.githack.com/w3c/wcag/findable_help_february_2021/understanding/22/findable-help.html) has been significantly updated, and the group believes it resolves the issue so this issue is being closed. There will be another review opened soon.
